nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals");
+const reservations = require("./data/reservations");
+const reviews = require("./data/reviews");
+
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("Application Result");
+});
+app.get("/meals", async (request, response) => {
+  // console.log(meals);
+  const allMealWithReview = meals.map((meal) => {
+    const mealReview = reviews.filter((review) => meal.id === review.mealId);
+    return {
+      ...meal,
+      review: mealReview,
```

Comment:
Be aware that you're writing "review" (singular) here, but in the corresponding test in the routes.spec.js file you're writing it as "reviews" (plural).

Reply:
Thanks for help. i got all issues solved. thanks